lib/zerigodns/host.rb

```diff
@@ -1,23 +1,24 @@
-class ZerigoDNS::Host < ZerigoDNS::Base
+class ZerigoDNS::Host < ZerigoDNS::Client
+  include ZerigoDNS::Resource
+  
+  base_path 'hosts'
+  resource_name 'host'
+  
+  
   class << self
     
     # Find host record(s) by zone and hostname
-    # @param [Symbol, #read] which One of :one, :first, :last, or :all.  See http://api.rubyonrails.org/v3.2.1/classes/ActiveResource/Base.html#method-c-find
     # @param [Zone, #read] zone The zone from which to find the host record.
     # @param [String, #read] hostname The hostname to find.
     # @return Host records, or an empty list if no records found.
-    def find_by_zone_and_hostname which, zone, hostname
```

Comment:
breaking change? Not sure if we should be sticking to full backwards compatibility here..